aspnetcore/release-notes/aspnetcore-10/includes/pipe-reader.md

```diff
@@ -1,16 +1,17 @@
-### Json+PipeReader deserialization support in MVC and Minimal
+### Json+PipeReader deserialization support in MVC and Minimal APIs
 
 PR: https://github.com/dotnet/aspnetcore/pull/62895
 
 See https://github.com/dotnet/core/blob/dotnet10-p7-libraries/release-notes/10.0/preview/preview7/libraries.md#pipereader-support-for-json-serializer
 
 MVC, Minimal APIs, and the `HttpRequestJsonExtensions.ReadFromJsonAsync` methods have all been updated to use the new Json+PipeReader support without requiring any code changes from applications.
 
-For the majority of applications this should have no impact on behavior. However, if the application is using a custom `JsonConverter`, there is a chance that the converter doesn't handle [Utf8JsonReader.HasValueSequence](https://learn.microsoft.com/dotnet/api/system.text.json.utf8jsonreader.hasvaluesequence) correctly. This can result in missing data and errors like `ArgumentOutOfRangeException` when deserializing.
+For most applications, the addition of this support has no effect on their behavior. However, if the application is using a custom `JsonConverter`, there's a chance that the converter doesn't handle [Utf8JsonReader.HasValueSequence](https://learn.microsoft.com/dotnet/api/system.text.json.utf8jsonreader.hasvaluesequence) correctly. This can result in missing data and errors like `ArgumentOutOfRangeException` when deserializing.
```

Comment:
```suggestion
For most applications, the addition of this support has no effect on their behavior. However, if the application is using a custom <xref:System.Text.Json.Serialization.JsonConverter>, there's a chance that the converter doesn't handle <xref:System.Text.Json.Utf8JsonReader.HasValueSequence%2A?displayProperty=nameWithType> correctly. This can result in missing data and errors like <xref:System.ArgumentOutOfRangeException> when deserializing.
```